VagrantProvision.sh

```diff
@@ -552,11 +552,8 @@ rm -rf $HOME/tmp
 
 cd $HOOT_HOME
 
-rm -rf $HOOT_HOME/ingest
-mkdir -p $HOOT_HOME/ingest/processed
-
-rm -rf $HOOT_HOME/upload
-mkdir -p $HOOT_HOME/upload
+rm -rf $HOOT_HOME/userfiles/ingest/processed
+mkdir -p $HOOT_HOME/userfiles/ingest/processed
 
```

Comment:
I will remove from the script:
-rm -rf $HOOT_HOME/upload
-mkdir -p $HOOT_HOME/upload

<Context docBase="/home/vagrant/hoot/userfiles/ingest/processed" path="/static" /> in server.xml
needs $HOOT_HOME/userfiles/ingest/processed to exist.  Otherwise, Tomcat fails to start.